src/transactions/reducer.ts

```diff
@@ -316,3 +316,23 @@ export const completedTxHashesByNetworkIdSelector = createSelector(
     return hashesByNetwork
   }
 )
+
+export const pendingStandbyTxHashesByNetworkIdSelector = createSelector(
```

Comment:
this is fine but we do also have `pendingStandbyTransactionsSelector`, it's less efficient to use i guess, but if the array is always small then i wonder if it's worth having 2 selectors that do basically the same thing?